src/org/opendatakit/briefcase/ui/SettingsPanel.java

```diff
@@ -83,14 +85,20 @@ public SettingsPanel(MainBriefcaseWindow parentWindow) {
         chkParallel.setSelected(BriefcasePreferences.getBriefcaseParallelPullsProperty());
         chkParallel.addActionListener(new ParallelPullToggleListener());
 
+        lblTrackingConsent = new JLabel(MessageStrings.TRACKING_CONSENT);
```

Comment:
I'd hoped it would as straightforward as giving each label an ActionListener, but they don't have a method to set one as such.